### PR TITLE
Use Go Modules

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

Switch from Glide to [Go modules](https://blog.golang.org/using-go-modules). **NOTE:** This requires you to re-clone the project _outside of your `GOPATH`_.

Addresses https://jira.coreos.com/browse/CONSOLE-1489
Resolves #1140

### CODE REVIEW

**Comment by spadgett** [comment]
We probably need to update the go version in our Dockerfiles. Just confirming dependencies are still vendored in?

**Comment by benjaminapetersen** [comment]
Might be helpful to split this into 2 commits:  changes, vendor.  Just to make it a bit easier to review.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module github.com/openshift/console
+
+go 1.12
+
```
Comment (benjaminapetersen) [inline go.mod]: How nice that it doc's the version of go that is used :)

**Comment by alecmerdler** [comment]
@benjaminapetersen Rebased.

**Comment by benjaminapetersen** [comment]
`Dockerfile.builder` is still `FROM golang:1.10-stretch`, bump to match `go 1.11+,`?

**Comment by alecmerdler** [comment]
> `Dockerfile.builder` is still `FROM golang:1.10-stretch`, bump to match `go 1.11+,`?

Fixed.

**Comment by spadgett** [comment]
> +210,548 −82,522

I wouldn't expect so many more lines added. Are we vendoring transitive dependencies when we weren't before?

It might be a non-issue. Just making sure we understand.

**Pull request merged**